src/transformers/tokenization_utils_base.py

```diff
@@ -0,0 +1,1775 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Base classes common to both the slow and the fast tokenization classes:
+    PreTrainedTokenizerBase (host all the user fronting encoding methodes)
+    Special token mixing (host the special tokens logic) and
+    BatchEncoding (wrap the dictionnary of output with special method for the Fast tokenizers)
```

Comment:
```suggestion
    BatchEncoding (wrap the dictionary of output with special method for the Fast tokenizers)
```

Reply:
If only French and English could agree on the number of n's they use, then that'd remove some confusion. 😄